org.eclipse.jdt.ls.core/plugin.xml

```diff
@@ -54,6 +54,8 @@
          point="org.eclipse.jdt.ls.core.delegateCommandHandler">
       <delegateCommandHandler class="org.eclipse.jdt.ls.core.internal.JDTDelegateCommandHandler">
             <command id="java.edit.organizeImports"/>
+            <command id="java.classpath.getChildren"/>
+            <command id="java.classpath.getSource"/>
```

Comment:
why do you need a getSource command? if objects have a source uri, then the client should just open that URI, whether it's a file uri or a jdt one

Reply:
Okay, for classfile it can direclty open in using URI. How about the JarEntryFile? What is the URI format of the JarEntryFile?

Reply:
Okay, fount the ContentProvider. I'll add a ContentProvider for JarEntryFile